supervisor/api/const.py

```diff
@@ -9,9 +9,6 @@
 
 COOKIE_INGRESS = "ingress_session"
 
-HEADER_TOKEN_OLD = "X-Hassio-Key"
-HEADER_TOKEN = "X-Supervisor-Token"
-
```

Comment:
These were moved up to the base `const` file. I had to do this because when in here they created an import loop (bootstrap -> utils/sentry -> misc/filter -> api -> unfinished coresys).

The alternatives are:
1. moving everything in `api/__init__.py` out to a different file so importing from `api/const` doesn't load it
2. removing the `CoreSys` typing in `utils/sentry` and `misc/filter` so `CoreSys` isn't imported along the way.

This was obviously the quickest fix and kind of made sense since the constants are used in `misc/filter` too. But I can take one of the other options instead if we prefer.